frontend/workflows/projectCatalog/src/catalog/index.tsx

```diff
@@ -60,19 +80,15 @@ const Catalog: React.FC<WorkflowProps> = ({ heading }) => {
     );
   };
 
-  const autoComplete = async (search: string): Promise<any> => {
-    // Check the length of the search query as the user might empty out the search
-    // which will still trigger the on change handler
-    if (search.length === 0) {
-      return { results: [] };
-    }
-
-    const response = await client.post("/v1/resolver/autocomplete", {
-      want: `type.googleapis.com/clutch.core.project.v1.Project`,
-      search,
-    });
+  const { handleSubmit } = useForm({
+    mode: "onSubmit",
+    reValidateMode: "onSubmit",
+    shouldFocusError: false,
+  });
 
-    return { results: response?.data?.results || [] };
+  const handleChanges = event => {
+    setCustomProject(event.target.value);
```

Comment:
I dont think you should need the extra customProject state, setting the value equal to `state.search` should be sufficient as long as the dispatch is correct.

Reply:
Sounds good let me remove it and see how it does.

Reply:
Seems to work just fine, i removed the extra state.